pegasus/sites.v3/code.org/views/workshop_search.haml

```diff
@@ -1,4 +1,5 @@
-%script{type: "text/javascript", src: "https://maps.googleapis.com/maps/api/js?client=#{CDO.google_maps_client_id}&sensor=true&libraries=places,geometry&v=3.7"}
+%script{type: "text/javascript", src: "https://maps.googleapis.com/maps/api/js?client=#{CDO.google_maps_client_id}&libraries=places,geometry&v=3.27"}
+%script{type: "text/javascript", src: "https://developers.google.com/maps/documentation/javascript/examples/markerclusterer/markerclusterer.js"}
```

Comment:
similarly, this doesn't seem like a valid production url. Should we just add this to the repo?